csstest.js

```diff
@@ -401,6 +404,83 @@ function passclass(info) {
 	return classes[index];
 }
 
+function resetOutput() {
+	mainScore = new Score();
+	_bTestResults = {};
+
+	// Output current score
+	ele('score').textContent = '';
+	ele('passedTests').textContent = '';
+	ele('totalTests').textContent = '';
+	ele('total').textContent = '';
+	ele('specsTested').textContent = '';
+	ele('all').textContent = '';
+
+	// Display time taken
+	ele('timeTaken').textContent = '';
+}
+
+function runTests(filter = '') {
+	let specs = [];
```

Comment:
I see you replaced some `var` keywords by `let`.
Do we want to embrace ES6-style javascript now? or still being somehow compatible with old browsers?

Reply:
IE isn't really dead yet. For me, it would be necessary to wait until it's definitely dead (removal of Windows). I let you decide.

Reply:
According to [caniuse.com](https://caniuse.com/let), IE has (partial) support for `let`. The biggest percentage of a non-supporting browser is Opera Mini with ~1%. And the overall support is at ~95%, which seems pretty stable to me.

Personally, I prefer to use ES6+ features as long as there is good browser support for them. If you two believe we should strive for full backwards compatibility, I could change it back to `var`.
Though I have to admit, I never tried to call the site in a very old browser so far. I assume there are other pieces of code that might be unsupported in older browsers.

Having said that, I am also planning to push #57 forward which requires a good browser coverage. So maybe that requires the JS part to be backwards compatible.

Sebastian

Reply:
> For me, it would be necessary to wait until it's definitely dead (removal of Windows).

For what it's worth, IE is removed in Windows 11 and finally [will be gone in June this year](https://www.microsoft.com/en-us/download/internet-explorer.aspx), though it will surely still take many years until nobody uses IE anymore.
According to [statcounter](https://gs.statcounter.com/browser-market-share) IE still has a market share of 0.46%.

Sebastian

Reply:
Because there's a mixture of `var` and `let` after that patch, I decided to roll back to `var` again for now.

I'll file a follow-up issue on refactoring the code base, instead.

Sebastian

Reply:
I've filed #216 now to discuss whether and which ES6+ features we want to use. Feel free to comment there!

Sebastian

Reply:
Could this line also be reverted to `var`?

Reply:
Done. Thank's for the hint!